src/pal/src/file/file.cpp

```diff
@@ -68,7 +74,10 @@ CObjectType CorUnix::otFile(
                 FileCleanupRoutine,
                 NULL,   // No initialization routine
                 0,      // No immutable data
+                NULL,
```

Comment:
Could you please add comments to all places where you have added NULLs for the new parameters describing what the parameter is (like it is done for the pre-existing parameters of the similar kind)? It would make reading the code easier.